ivy/utils/exceptions.py

```diff
@@ -288,6 +290,11 @@ def __init__(self, message=""):
         super().__init__(message)
 
 
+class InplaceUpdateException(IvyException):
```

Comment:
let's move this exception class at the end of the other exception classes and before `handle_exceptions`